test/stomp_handler_test.dart

```diff
@@ -287,7 +287,10 @@ void main() {
       handler = StompHandler(
         config: config.copyWith(
           onConnect: (StompFrame frame) {
-            handler!.ack(id: 'message-0', headers: {'receipt': 'send-0'});
+            handler!.ack(
+                id: 'message-0',
+                headers: {'receipt': 'send-0'},
+                ackHeaderKey: '');
```

Comment:
there is no need to specify the ackHeaderKey since you gave it a default value, no?